tests/foundry/test/ContractAssert.t.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+
+contract ContractAssert {
+    function f() public pure {
+        assert(false);
+    }
+}
+
+contract ContractAssertTest is Test {
+    ContractAssert c;
+
+    function test_call_empty_account() public view{
+        c.f();
+    }
```

Comment:
I don't think I understand this test. Is teh idea that because the contract was not deployed with `new ContractAssert()`, the account is actually starting as empty, and so it does not execute the code with `assert(false)`?